modules/contact/src/ContactSplit.C

```diff
@@ -27,6 +27,9 @@ InputParameters validParams<ContactSplit>()
   params.addParam<std::vector<std::string> >("uncontact_master", "Master surface list for excluded contacts");
   params.addParam<std::vector<std::string> >("uncontact_slave",  "Slave surface list for excluded contacts");
   params.addParam<std::vector<int> >("uncontact_displaced", "List of indicators whether displaced mesh is used to define excluded contact");
+  // Right now, we consider this as a required parameter.
+  // After some tests from BISON, we will set a default value for this parameter.
+  params.addRequiredParam<bool >("include_all_contact_nodes","Whether to include all nodes on the contact surfaces");
```

Comment:
Why the space after `bool`? You only need spaces between multiple right angle brackets for C++03 compilers. We don't want space around the types.